appendices.tex

```diff
@@ -370,6 +370,152 @@ \chapter{Listing of application roles}
 \item [yearend\_run]
 \end{description}
 
+\chapter{Deprecated Content}
+\label{app-deprecated-content}
+
+\section{ledgersmb.conf}

```

Comment:
Since this section is deprecated and therefor shouldn't change anymore, maybe it's ok to copy the descriptions from https://github.com/ledgersmb/LedgerSMB/blob/master/doc/conf/ledgersmb.conf.default without linking them yet (the linking having been discussed in the chat channel).

Reply:
Ok, working on it.

Reply:
I left the deleted items as comments so you can review.